cfme/tests/openstack/infrastructure/test_roles.py

```diff
@@ -67,6 +65,7 @@ def test_roles_summary(provider, soft_assert):
             soft_assert(res, err_ptrn.format(v))
 
 
+@pytest.mark.trylast
```

Comment:
@okolisny Although it doesn't break anything in test execution, we ask that you remove it anyway. One thing we would like to avoid is contributors coming after you reading such decorators and thinking it appropriate to use dependence.